src/components/Dropdown/index.jsx

```diff
@@ -0,0 +1,60 @@
+// @flow
+import React, { useState, useCallback } from "react";
+import * as _ from "lodash";
+import classNames from "classnames";
+
+type Props = {
+  initValue: $ReadOnly<{ value: string, label: string }>,
+  label: string,
+  items: $ReadOnlyArray<{ value: string, label: string }>
+};
+
+export default function Dropdown(props: Props) {
+  const { initValue, label, items } = props;
+  const [isExpanded, setIsExpanded] = useState(false);
+  const [selectedItem, setSelectedItem] = useState(initValue);
+  const expandMenu = useCallback(() => {
+    setIsExpanded(isShown => !isShown);
+  }, [setIsExpanded]);
+  const selectItem = useCallback(
+    item => {
+      setIsExpanded(false);
+      setSelectedItem(_.find(items, i => i.value === item.value));
+    },
+    [setIsExpanded, setSelectedItem]
+  );
+
+  return (
+    <div className="gc-dropdown">
+      <label className="gc-dropdown__label" htmlFor="gc-dropdown__label">
+        {label}
+      </label>
+      <div className="gc-dropdown__menu">
+        <button className="gc-dropdown__btn" onClick={expandMenu}>
+          {selectedItem.label}
+        </button>
+        {isExpanded && (
+          <div className="gc-dropdown__content">
+            <ul className="gc-dropdown__list">
+              {/* eslint-disable jsx-a11y/no-static-element-interactions */}
+              {_.map(items, item => {
+                return (
+                  <li
+                    className={classNames("gc-dropdown__item", {
```

Comment:
Assign `classNames` to a variable (ex `classes`) first. Also, you're comparing object references, I don't think its a good approach, you can check is index value equal to `0` - that'll mean we're dealing with very first element.